index.html

```diff
@@ -0,0 +1,154 @@
+<html>
+	<head>
+		<meta name="viewport" content="width=device-width, intial-scale=1.0">
+		<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
+		<link href="style.css" rel="stylesheet" type="text/CSS">
+		<title >Robin Hodge's Portfolio Website</title>
+	</head>
+	<body id="top">
+		<div class="contact">
+			<div class="contact-outer-container contact-top">
+				<h2>Contact</h2>
+				<p> The best way to get in touch with me is via email, but if you prefer social media I have included these too.</p>
+				<div class="contact-container">
+					<a href="mailto:mrrobinhodge@gmail.com"><img src="email.png" alt="An email thumbnail"></a>
```

Comment:
a more conventional alt for this would be "Email icon" because thumbnail means something else (same for the other ones)

Reply:
done!